chain-api/src/utils/dex/tick.helper.ts

```diff
@@ -132,6 +135,16 @@ export function flipTick(bitmap: Bitmap, tick: number, tickSpacing: number) {
   bitmap[word] = newMask.toString();
 }
 
+function isTickInitialized(tick: number, tickSpacing: number, bitmap: Bitmap): boolean {
+  tick /= tickSpacing;
+  const [word, pos] = position(tick);
+  const mask = BigInt(1) << BigInt(pos);
+  if (bitmap[word] === undefined) return false;
+  const currentMask = BigInt(bitmap[word]);
+  const newMask = currentMask ^ mask;
+  return newMask == BigInt(0);
```

Comment:
```suggestion
  const mask = BigInt(1) << BigInt(pos);

  if (bitmap[word] === undefined) return false;

  const currentMask = BigInt(bitmap[word]);
  const newMask = currentMask ^ mask;

  return newMask == BigInt(0);
```

Newlines improve readability around the one-line if statement.